packages/worker/src/debug-replayer/index.ts

```diff
@@ -0,0 +1,75 @@
+import worker_threads from 'node:worker_threads';
+import { temporal } from '@temporalio/proto';
+import { ReplayWorkerOptions } from '../worker-options';
+import { Worker } from '../worker';
+import { Client } from './client';
+
+async function run(options: ReplayWorkerOptions): Promise<void> {
+  const baseUrl = process.env.TEMPORAL_DEBUGGER_PLUGIN_URL;
+  if (!baseUrl) {
+    throw new Error('Missing TEMPORAL_DEBUGGER_PLUGIN_URL environment variable');
+  }
+  const client = new Client({ baseUrl });
+  const response = await client.get('history');
+  const history = temporal.api.history.v1.History.decode(
+    await Client.readAll(response),
+    Client.contentLength(response)
+  );
+
+  const thread = new worker_threads.Worker(require.resolve('./worker-thread'));
+
+  const rejectedPromise = new Promise<void>((_, reject) => {
+    // Set the global notifyRunner runner function that can be used from the workflow interceptors.
+    // The function makes an HTTP request to the runner in a separate thread and blocks the current thread until a
+    // response is received.
+    (globalThis as any).notifyRunner = (wftStartEventId: number) => {
+      const sab = new SharedArrayBuffer(4);
+      const responseBuffer = new Int32Array(sab);
+      thread.postMessage({ eventId: wftStartEventId, responseBuffer });
+      Atomics.wait(responseBuffer, 0, 0);
+      if (responseBuffer[0] === 2) {
+        // Error occurred (logged by worker thread)
+        reject(new Error('Failed to call runner back'));
+      }
+    };
+  });
+
+  await Promise.race([
+    rejectedPromise,
+    Worker.runReplayHistory(
+      {
+        ...options,
+        interceptors: {
+          ...options.interceptors,
+          workflowModules: [
+            // Inbound goes first so user can set breakpoints in own inbound interceptors.
+            require.resolve('./debug-inbound-interceptor'),
```

Comment:
Filename does match that of commited files... => `inbound-interceptor.ts`?

Reply:
Thanks... it worked locally because I had compiled versions of these files before the rename.